common_test.go

```diff
@@ -74,6 +85,13 @@ func (s *BaseSuite) NewRepositoryFromPackfile(f *fixtures.Fixture) *Repository {
 		panic(err)
 	}
 
+	storer.SetReference(plumbing.NewHashReference(plumbing.HEAD, f.Head))
+
+	r, err := Open(storer, memoryfs.New())
```

Comment:
In this case you are opening a storer that is not empty, so for me Open is correct